pygccxml/binary_parsers/undname.py

```diff
@@ -187,8 +187,9 @@ def format_argtypes(self, argtypes, hint):
                 # ugly hack, later, I will replace ', ' with ',', so single
                 # space will still exist
                 argsep = ',  '
-            return argsep.join(map(self.__format_type_as_undecorated(
-                type_, True, hint), argtypes))
+            return argsep.join(
+                map(lambda type_: \
+                    self.__format_type_as_undecorated(type_, True, hint), argtypes))
```

Comment:
Not sure I understand this code 100%, but I think the intent was to pass a lambda to map, and `type_` is supposed to be the name of the lambda argument.